src/systems/environment-system.js

```diff
@@ -34,7 +34,13 @@ const defaultEnvSettings = {
   physicallyCorrectLights: true,
   envMapTexture: null,
   backgroundTexture: null,
-  backgroundColor: new THREE.Color("#000000")
+  backgroundColor: new THREE.Color("#000000"),
```

Comment:
We are using different default values in Spoke, Blender and here. We should decide reasonable defaults and unify for consistency:

https://github.com/mozilla/Spoke/blob/d98a535b948a3d144290b87d459a39e978683d7f/src/editor/nodes/SceneNode.js#L151-L155

Maybe #FFFFFF and a default density of 0.00025.

Reply:
Ah these defaults were from the fog component in aframe. I will pull what we have in Blender.

Reply:
Actually yeah I think the `0.1` default we have in blender is probably way too high. Will use `0.00025` and we can update that in Blender as well.